### ISSUE TITLE
Regarding Training Effectiveness

### ISSUE THREAD

**Issue by Azer90** [clovaai/lffont#42]
Hello, I would like to ask if the effect shown in the image below is normal when I reached 550000 steps in the first stage of training using LFfont. Is it because the training was not done properly or is there a way to modify hyperparameters? Or is it not possible . Could you please help me check if the loss information for this training is normal? Thank you

![Image](https://github.com/user-attachments/assets/5af2b70c-a538-43c5-a31f-212dcd68a981)

![Image](https://github.com/user-attachments/assets/491139c5-4cdf-4732-8436-e4fa3d207b69)

![Image](https://github.com/user-attachments/assets/b57475da-6d17-48e1-8feb-d410009f3579)

![Image](https://github.com/user-attachments/assets/969f3225-4895-405a-9bb5-f7d2adc8794c)
